influxdb/line_protocol.py

```diff
@@ -16,6 +16,12 @@
 EPOCH = UTC.localize(datetime.utcfromtimestamp(0))
 
 
+def _to_nanos(timestamp):
+    delta = timestamp - EPOCH
+    nanos = (delta.days * 86400 + delta.seconds) * 10 ** 9 + delta.microseconds * 10 ** 3
```

Comment:
this line fails flake8 (89 chars), mind splitting it up? otherwise, this looks like a good change.